config/src/config/test_data/validator_full_node.config.yaml

```diff
@@ -0,0 +1,38 @@
+base:
+    data_dir: "/opt/libra/data/common"
+    role: "full_node"
+    waypoint:
+        type: "from_storage"
+        backend:
+            type: "vault"
+            server: "https://127.0.0.1:8200"
+            ca_certificate: "/full/path/to/certificate"
+            token:
+                type: "from_disk"
+                path: "/full/path/to/token"
+
+execution:
+    genesis_file_location: "relative/path/to/genesis"
+
+full_node_networks:
+    - listen_address: "/ip4/0.0.0.0/tcp/6180"
+      identity:
+          type: "from_storage"
+          key_name: "fullnode_network"
+          peer_id_name: "operator_account"
+          backend:
+              type: "vault"
+              server: "https://127.0.0.1:8200"
+              ca_certificate: "/full/path/to/certificate"
+              token:
+                  type: "from_disk"
+                  path: "/full/path/to/token"
+      network_id:
+          type: "public"
+    - listen_address: "/ip4/0.0.0.0/tcp/7180"
+      network_id:
+          type: "private"
+          name: "vfn"
+      seed_peers:
+          "c227da54069989f283712e4016704660":
+              - "/ip4/127.0.0.1/tcp/58259/ln-noise-ik/c998dcd54c3daf48e0ad516d94b7be0b0b7a27caa00541f2b2c14b13500df10b/ln-handshake/0"
```

Comment:
Is this to support multiple addresses per peer?

Reply:
I believe so. The onchain addresses support multiple addresses for when the client rotates... at which point if we just pull down all of them and store them to this seed_peers we would ideally have all of them since it would be unclear which one was the "best"